scripts/procesory.csv

```diff
@@ -0,0 +1,42 @@
+L.p.,Nazwa,Data,Bity,Złącze,Tranzystory (mln),Technologia (nm),Taktowanie (MHz),Tranzystory,,Na wystawie?,Stan,Zestaw,Informacje dodatkowe
```

Comment:
I think script should use xlsx, not csv.
This should be three lines more at most.

+ I think collections should be dynamically added depending on sheet name in Excel Worksheet.
It's not convenient to generate csv for every sheet, this should be done automatically.

Please consider.

Reply:
I can change it to work directly on xlsx, but I would be opposed to importing multiple sheets at once. For example, if the user wants to set a certain field as private when importing from one sheet, and as public when importing from another sheet, it would complicate everything a lot.

Dynamically adding and creating collections depending on sheet name doesn't sound like a bad idea, but I'm not sure if it's worth adding. Typing the name of a collection a few times isn't a huge amount of work for the user, and it adds a small extra level of control.

But directly reading xlsx sounds good. What do you think about adding a "--sheet_name" argument to let the user choose which sheet from xlsx file to import?

Reply:
1. Makes sense. Let's leave as is then.
2. `--add-collection`? If it's difficult to do please leave as is. Use your judgement ;)
3. yep makes sense. OR let's create another script which just saves all sheets to csv in a directory `exported_sheets` and then provide file here via arg so that this script is not overcomplicated that much. What do you think?